core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -73,14 +73,14 @@ object RequestChannel extends Logging {
     buffer = null
     private val requestLogger = Logger.getLogger("kafka.request.logger")
 
-    private def requestDesc: String = {
+    private def requestDesc(details: Boolean): String = {
```

Comment:
nit: ``requestDesc()`` is called two times with ``false`` and only one with ``true``. We could setup the default value to ``false`` as:
```
private def requestDesc(details: Boolean = false): String = {
```

But I myself admit this something so irrelevant that I don't know if there is a real gain. Up to you.